stl/src/filesys.cpp

```diff
@@ -53,7 +53,7 @@ static HANDLE _FilesysOpenFile(const wchar_t* _Fname, DWORD _Desired_access, DWO
         &_Create_file_parameters);
 #else // _CRT_APP
     return CreateFileW(
-        _Fname, _Desired_access, FILE_SHARE_READ | FILE_SHARE_WRITE | FILE_SHARE_DELETE, 0, OPEN_EXISTING, _Flags, 0);
+        _Fname, _Desired_access, FILE_SHARE_READ | FILE_SHARE_WRITE | FILE_SHARE_DELETE, nullptr, OPEN_EXISTING, _Flags, nullptr);

```

Comment:
The Code Format Validation step of the build is failing because clang-format wants to wrap this and several other lines differently, due to the 120 column limit configured in `.clang-format`. In the PR's checks, if you click on the Details link next to Code Format Validation, we've set up Azure Pipelines to display the affected files *and* the diffs that clang-format wants to apply. You can either run clang-format on those files, or simply apply the diffs manually.

Reply:
I hope my latest addresses this.